src/python/turicreate/test/test_sframe.py

```diff
@@ -216,6 +216,12 @@ def test_auto_parse_csv(self):
             self.assertEqual(sf.dtype, [float, int, str])
             self.__test_equal(sf, df)
 
+    def test_drop_duplicates(self): 
+        sf = SFrame({'A': ['a', 'b', 'a'], 'B': ['b', 'a', 'b'], 'C': [1, 2, 1]}) 
+        df1=sf.drop_duplicates(subset=['A']) 
+        sf1 = pd.DataFrame({'A': ['a','b'], 'B': ['b','a'], 'C': [1,2]}) 
+        self.__test_equal(df1,sf1) 
```

Comment:
Let's add a couple of random tests here too.  Take a look at tc.util.generate_random_sframe; I think this test doesn't cover all the functionality (e.g. multiple columns in the subset.  

I'd create a function that uses pandas to test equality, but you also need to e.g. sort them to make sure they are equal.  

E.g. test `sorted(list(pandas_result)) == sorted(list(turi_result))`.  Group_by doesn't always preserve order.